### ISSUE TITLE
Graphite not creating after working fine for years

### ISSUE THREAD

**Issue by joeg1484** [graphite-project/carbon#771]
Hi All,

We have had a Graphite install for a couple of years now and all of the sudden, on April 26th, I believe, it stopped writing to the whisper database. We can see historical data before that date, just nothing after is showing up.

We are using the Icinga2 connector and I see Icinga connected and sending data, but nothing in creates.log except the last couple entries when the log was rotated.

It worked fine for the whole time it was up and running, but now, its not writing data.

Any idea how to troubleshot or what I can look at? I tried restarting the services and everything connects back up like it should, yet, not whisper updates happening.

If you need to see a config file or log, let me know.

Server is running on:
Red Hat Enterprise Linux Workstation release 6.9 (Santiago)
Django version 1.4.21
graphite-web-0.9.16-1.el6.noarch
twistd 8.2.0
python-carbon-0.9.16-1.el6.noarch
$  graphite-manage --version
1.4.21

Thanks!
Joe

**Comment by deniszh** [comment]
Hi @joeg1484,

usually Graphite is quite robust and not stop working out of the blue. Please check do you have disk space, file permissions, please check whisper files with `whisper-info` - maybe they're corrupted.

**Comment by joeg1484** [comment]
Hi @deniszh ,

Thanks for the repy. Not sure what happend, but I think I might be over taxing the system. I noticed that in the icinga2 writer logs that it was taking a very long time (< 15 min) to finish transactions to the graphite writer. 

What I did was changed one of the settings in the carbon.conf (Dont have it in front of me atm) to = inf and restarted the system. Over about 30 min, stats started coming back in.

We are thinking about separating the graphite and icinga2 systems into 2 separate machines as not they are on the same VM :-P.

Thanks for the reply!
Joe
